app/flatpak-table-printer.c

```diff
@@ -208,25 +208,58 @@ flatpak_table_printer_finish_row (FlatpakTablePrinter *printer)
   printer->current = g_ptr_array_new_with_free_func (free_cell);
 }
 
-static void
-print_row (GString *row_s, gboolean bold)
+static int
+print_row (GString *row_s, gboolean bold, int *skip, int columns)
 {
+  int rows;
+  char *p, *end;
+
   g_strchomp (row_s->str);
-  if (bold)
-    g_print (FLATPAK_ANSI_BOLD_ON"%s"FLATPAK_ANSI_BOLD_OFF"\n", row_s->str);
-  else
-    g_print ("%s\n", row_s->str);
+  rows = 1 + (strlen (row_s->str) / columns);
```

Comment:
This isn't right. An 80 char row is only one row, not two. It seems like the position "after" the width of the terminal is special in that we don't wrap until you print another (non-newline) char. So, if you print 80 chars + newline, then that is just one row. (Tested in gnome-terminal and xterm).

Reply:
Seems like `(columns + len - 1) / columns` is right?